docker/Dockerfile

```diff
@@ -96,12 +101,12 @@ COPY --chown=model-server --from=compile-image /home/venv /home/venv
 
 ENV PATH="/home/venv/bin:$PATH"
 
-COPY dockerd-entrypoint.sh /usr/local/bin/dockerd-entrypoint.sh
+COPY docker/dockerd-entrypoint.sh /usr/local/bin/dockerd-entrypoint.sh
```

Comment:
are you expecting users to instead run this dockerfile from `serve` instead of `docker`?

Reply:
https://github.com/klae01/serve-dockerfile-update/blob/71e962cc2cb38e74da033ce5ed3e02434cb5906c/docker/build_image.sh#L111

I change provided `context` as `serve` (not only `docker`) while docker build.
so, still docker build at `docker`